python/tests/test_logistic_regression.py

```diff
@@ -109,7 +112,9 @@ def test_toy_example(gpu_number: int) -> None:
         schema = features_col + " array<float>, " + label_col + " float"
         df = spark.createDataFrame(data, schema=schema)
 
-        lr_estimator = LogisticRegression(regParam=1.0, num_workers=gpu_number)
+        lr_estimator = LogisticRegression(
+            standardization=False, regParam=1.0, num_workers=gpu_number
```

Comment:
Do we have a test comparing objective or coefficients to cpu spark that would have failed before given that spark has standardization on by default?

Reply:
Yes, the PR compares coefficients. There is a compare_model to assert CPU and GPU having same coefficients. The compare_model is used everywhere in the test cases of this PR when testing standardization=True.

The test_compat function that was just revised shows the coefficients are different when standardization is set to True and False. GPU and CPU match in both cases.

Regarding objective, the test_sparse_nlp20news compares that in both cases when standardization = True and False.